### ISSUE TITLE
No custom ldif import when replication.enabled is set to false

### ISSUE THREAD

**Issue by zirrok80** [jp-gouin/helm-openldap#87]
Hi. We were facing some problems/bugs upgrading from 3.0.2 to 4.0.2. The first one:

The schema files (e.g. syncprov.ldif, bsyncprov.ldif, ...) are only mounted if replication.enabled is true. But they are hardcoded in LDAP_EXTRA_SCHEMAS. So when replication.enabled is set to false these fiies cant be found and due to an error the pod will be restarted before reaching the import of custom ldif files defined via customLdifFiles. 
After the restart the evaluation of !is_dir_empty "$LDAP_DATA_DIR"; in libopenldap.sh is true, so that the custom ldif files will never be imported.

In short: replication.enabled = false  =>  no import of custom ldif files

The workaround for us was to override the LDAP_EXTRA_SCHEMAS env variable.

Best regards
Björn

**Comment by jp-gouin** [comment]
Hi @zirrok80 ,

Indeed right now `LDAP_EXTRA_SCHEMAS ` is harcoded , where it should be a variable in the values and computed into the env variable .

feel free to submit a PR if you feel like it .

it will be available in the next release

**Comment by jp-gouin** [comment]
Fixed in the last version `4.1.0`
